api_tests/revisions/views/test_registrations_schema_responses_list.py

```diff
@@ -0,0 +1,108 @@
+import pytest
+
+from osf_tests.factories import (
+    SchemaResponsesFactory,
+    RegistrationFactory,
+    RegistrationSchemaFactory,
+    AuthUserFactory,
+)
+from django.contrib.contenttypes.models import ContentType
+from osf.models.schema_responses import SchemaResponses
+
+
+@pytest.mark.django_db
+class TestRegistrationsSchemaResponseList:
+
+    @pytest.fixture()
+    def non_contrib(self):
+        return AuthUserFactory()
+
+    @pytest.fixture()
+    def user(self):
+        return AuthUserFactory()
+
+    @pytest.fixture()
+    def user_write(self):
+        return AuthUserFactory()
+
+    @pytest.fixture()
+    def user_admin(self):
+        return AuthUserFactory()
+
+    @pytest.fixture()
+    def registration(self, schema):
+        return RegistrationFactory()
+
+    @pytest.fixture()
+    def payload(self, registration):
+        return {
+            'data':
+                {
+                    'type': 'registrations',
```

Comment:
This should be "revisions", the type of the relationship entity should be "registrations"

Reply:
I am going off the API contract, so not sure what is correct.